packages/twenty-server/src/modules/workflow/common/exceptions/workflow-query-validation.exception.ts

```diff
@@ -1,8 +1,12 @@
 import { CustomException } from 'src/utils/custom-exception';
 
 export class WorkflowQueryValidationException extends CustomException {
-  constructor(message: string, code: WorkflowQueryValidationExceptionCode) {
-    super(message, code);
+  constructor(
+    message: string,
+    code: WorkflowQueryValidationExceptionCode,
+    { displayedErrorMessage }: { displayedErrorMessage?: string } = {},
```

Comment:
why is displayedErrorMessage an object here whereas message is not? makes the API asymetrical without any strong reason?